.env.dist

```diff
@@ -0,0 +1,27 @@
+# This file is a "template" of which env vars need to be defined for your application
+# Copy this file to .env file for development, create environment variables when deploying to production
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=33fc24e575b5b402d5aec5ffd335539b
+#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
+#TRUSTED_HOSTS=localhost,example.com
+###< symfony/framework-bundle ###
+
+# mongo
+PAYUM_MONGO_PORT=27018
+PAYUM_MONGO_DATABASE=payum_server
+PAYUM_MONGO_URI=mongodb://mongo/${PAYUM_MONGO_DATABASE}
+
+# mongo test
```

Comment:
.test.env - move all test related envs to this file. 
load these vars in boostrap.php using Symfony Env component

Reply:
Done.